server/pulp/server/async/app.py

```diff
@@ -30,6 +31,92 @@
 _logger = logging.getLogger(__name__)
 
 
+class HeartbeatStep(bootsteps.StartStopStep):
+
+    """
+    This class is a celery "Blueprint". It extends the functionality of the celery
```

Comment:
Can a 1-line summary description be added, and then a whitespace. See the reference for multi-line docstrings here:  https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings